src/Refactoring-Core/RBFindAndReplaceRefactoring.class.st

```diff
@@ -0,0 +1,192 @@
+Class {
```

Comment:
Can we have a comment?
Now we will let it like that but I wonder if this is not just a transformation. 
From that perspective I would like to see if Refactoring2 cannot help us there.